portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDMTests.testcase

```diff
@@ -0,0 +1,94 @@
+@component-name = "portal-document-management"
+definition {
+
+	property testray.main.component.name = "Document Management";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+
+		task ("Given the DM Portlet is added to the page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "DM Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				column = 1,
+				groupName = "Guest",
+				layoutName = "DM Page",
+				widgetName = "Documents and Media");
+
+			Navigator.gotoPage(pageName = "DM Page");
+
+			LexiconEntry.changeDisplayStyle(displayStyle = "list");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			DMDocument.tearDownCP();
+
+			DMFolder.tearDownCP();
+
+			PagesAdmin.tearDownCP();
+		}
+	}
+
+	@description = "Check that is possible to download a Document and Media"
+	@priority = 5
+	test CanDownloadDM {
+		var dmEntryTitle1 = "DMTest 1";
+		var dmEntryFile1 = "Document_1.txt";
+		var dmEntryTitle2 = "DMTest 2";
+		var dmEntryFile2 = "Document_2.doc";
+
+		task ("And create 2 Documents and Media (Upload the file)") {
+			JSONDocument.addFileWithUploadedFile(
+				dmDocumentTitle = ${dmEntryTitle1},
+				groupName = "Guest",
+				mimeType = "text/plain",
+				sourceFileName = ${dmEntryFile1});
+
+			JSONDocument.addFileWithUploadedFile(
+				dmDocumentTitle = ${dmEntryTitle2},
+				groupName = "Guest",
+				mimeType = "application/msword",
+				sourceFileName = ${dmEntryFile2});
+
+			Refresh();
```

Comment:
Hey, can we try add for loops here too? Thanks!

Reply:
Okay, will do. Thank you for the review!